system/Pager/PagerRenderer.php

```diff
@@ -282,20 +289,18 @@ public function getCurrent(): string
 	 */
 	public function links(): array
 	{
-		$links = [];
-
```

Comment:
if it going to happen, it needs return early like this:

if $this->links !== [] 
{
    return $this->links; 
}